### PR TITLE
Added support for accepting `jscodeshift` ignore options (e.g. `--ignore-config` and `--ignore-pattern`)

### PR DESCRIPTION
**Pull request by vinomanick**
#88

### CODE REVIEW

File: src/options-support.js [hunk 1]
```diff
@@ -20,4 +22,18 @@ function getOptions() {
...
+  codeShiftOptions.forEach(option => {
+    if (Object.prototype.hasOwnProperty.call(cliOptions, option)) {
+      let camelCaseOption = option.replace(/-([a-z])/g, (_, up) => up.toUpperCase());
+      jsCodeShiftOptions.push(`--${option}`, options[option]);
+      delete cliOptions[option];
+      delete cliOptions[camelCaseOption];
+    }
+  });
+  return { cliOptions, jsCodeShiftOptions };
+}
```
Comment (rwjblue) [inline src/options-support.js:37]: I think I'd prefer to do this all in `parseTransformArgs`, and ahve that essentially `return { paths, options, transformerOptions }` or something like that.

What do you think?

Comment (vinomanick) [reply src/options-support.js]: Hi, this is my first open-source contribution that's why I was little hesitant towards altering the existing function and breaking the existing flow. Anyways, I have incorporated the changes in the parseTransformArgs itself.
Kindly have a look and let me know if this is okay.

**Comment by rwjblue** [comment]
I just landed a fairly big refactor from @simonihmig, would you mind rebasing on top of that work? I _think_ it shouldn't dramatically affect what you have done here, but I just want to make sure CI is happy with the two changes together before landing.

Also, we need to probably figure out how to do the parsing a bit better in general (instead of using `yargs.parse()`, we could provide the explicit options or something), but that is definitely an issue for another time...

**Comment by vinomanick** [comment]
I have rebased with the master and things look good. Will try to create a new PR to address the parsing.

**Pull request merged**